packages/www/layouts/streamDetail.tsx

```diff
@@ -268,6 +268,33 @@ const StreamDetail = ({
   const globalIngestUrl = `rtmp://rtmp.livepeer.${domain}/live`;
   const globalPlaybackUrl = `https://cdn.livepeer.${domain}/hls/${playbackId}/index.m3u8`;
 
+  const videoJsOptions = {
+    autoplay: true,
+    controls: true,
+    responsive: true,
+    fluid: true,
+    sources: [
+      {
+        src: globalPlaybackUrl,
+        type: "application/x-mpegURL",
+      },
+    ],
+  };
+
+  const handlePlayerReady = (player) => {
+    playerRef.current = player;
+
+    player.hlsQualitySelector();
```

Comment:
Maybe this could be default, always done from the player itself 🤔 
Will we ever want not to have this?

Reply:
Yeah good call. I made it a default.